public/css/base.css

```diff
@@ -0,0 +1,3 @@
+img {
+  width: 500px;
```

Comment:
This is just a placeholder rule because the placeholder image we used is 1200 px wide